src/userJourney/userJourneyConstants.ts

```diff
@@ -0,0 +1,23 @@
+import { IThresholdTier, IUserJourneyThresholds } from "../types";
+export const USER_JOURNEY_THRESHOLDS: IUserJourneyThresholds = {
```

Comment:
These should probably live in this file https://github.com/Zizzamia/perfume.js/blob/master/src/vitalsScore.ts